alvr/client_core/src/graphics/stream.rs

```diff
@@ -67,7 +68,9 @@ impl StreamRenderer {
         let shader_module =
             device.create_shader_module(include_wgsl!("../../resources/stream.wgsl"));
 
-        let mut constants = HashMap::from([
+        let mut constants = HashMap::new();
+
+        constants.extend([
```

Comment:
lol

Reply:
I did this for symmetry. first I create the hashmap, then I extend with each group of constants